ui/src/pages/backoffice/DocumentRequest/DocumentRequestDetails/components/DocumentRequestSteps/components/ProviderStep/ProviderStep.js

```diff
@@ -9,30 +18,60 @@ import {
   serializeBorrowingRequest,
 } from '@components/ESSelector/serializer';
 import { goTo } from '@history';
-import { STEPS } from '../DocumentRequestSteps';
+import { invenioConfig } from '@config';
+import { STEPS } from '../../DocumentRequestSteps';
+
+export const ProviderStep = ({ step }) => (
+  <Step active={step === STEPS.provider} disabled={step === STEPS.document}>
+    <Icon name="truck" />
+    <Step.Content>
+      <Step.Title>Select Provider</Step.Title>
+      <Step.Description>
+        Acquire or borrow from another Library
```

Comment:
Can they only acquire from other libraries or can they also acquire from another source?

Reply:
@equadon they can from another source (i.e. vendor), but then its not an ILL request but an acquisition. `Acquire` is very misleading in the particular case.